rpa-builder/6.3/modules/ROOT/pages/toolbox-flow-control-file-controlled-loop.adoc

```diff
@@ -2,105 +2,107 @@
+
+The first line of the file is considered the header line. _File Controlled Loop_ uses these headers as variable names and not as actual variable values.
 
 == Properties
 
 * *External File*
-** *Import Filename* The full path of the file that is imported
-** *Relative to 3rd Party Folder* Decides whether or not the file should
-be loaded from the 3rd party folder
-** *Shuffle* Shuffles the data rows during execution
+** *Import Filename* 
++
+Full path of the file to import.
+** *Relative to 3rd Party Folder* 
++
+Specifies if the path of the file to load is relative to the _3rd party_ folder.
+** *Separation character*
++
+Specifies the character to use as a column separator when reading the CSV file.
+** *Shuffle* 
++
+Shuffles the data rows during execution.
 * *Miscellaneous*
-** *Iteration Count* If _Limit iterations_ is selected, this limits the
-maximum iteration count used by the loop. Only applicable when the file
+** *Iteration Count* 
++
+If *Limit iterations* is selected, this property specifies the
+maximum iterations the loop does. Only applicable when the file
 is imported at runtime.
-** *Limit iterations* Decides whether iteration count is limited or not.
+** *Limit iterations* 
++
+Decides whether the iteration count is limited or not.
 Only applicable when the file is imported at runtime.
-** *Separate File Path for Execution* When _Use separate File Path for
-Execution_ is selected the file will be read from this path instead of
-the path given in _Import Filename_. Only applicable when the file is
+** *Separate File Path for Execution* 
++
+When *Use separate File Path for Execution* is selected, _File Controlled Loop_ reads the file from this path instead of
+the path given in *Import Filename*. Only applicable when the file is
 imported at runtime.
-** *Use separate File Path for Execution* Select this setting to load
-the file from a different path during execution. Useful if you want to
+** *Use separate File Path for Execution* 
++
+Select this setting to load the file from a different path during execution. Useful if you want to
 read the file from a network location. Only applicable when the file is
 imported at runtime.
-** *Use type string only* Converts *all* data to type String during
-runtime. In the wizard you can convert the data per column.
+** *Use type string only* 
++
+Converts all data to type String during runtime. In the wizard, you can convert the data per column.
 
 == Inbound Variables
 
-The Action Step File Controlled Loop has no properties.
+The Action Step File Controlled Loop has no Inbound variables.
 
 == Outbound Variables
 
-Within the _Action Steps File Controlled Loop_ the respective column
-contents are returned as variables.
+The Action Steps inside _File Controlled Loop_ return the respective column

```

Comment:
Before, you tagged the whole Action Steps File Controlled Loop and now no. If it's not part of the UI, then Action Steps should be action steps. If it's fine as is, ignore this comment.